### PR TITLE
Added Lifestreet adapter

### PR DESCRIPTION
**Pull request by Lifestreet**
A Lifestreet adapter for Prebid Server. 
Test parameters:
`[
  {
    "bidder": "lifestreet",
    "params": {
      "slot_tag": "slot178682.159"
    }
  }
]`

### CODE REVIEW

**Comment by nhedley** [comment]
This looks good to me. Got a test ad back using test parameters. 

@mkendall07 Can you take a final look at this?

**Comment by vodianyk** [comment]
Hey guys! Could you please let us know when we can expect this merging?

**Comment by Lifestreet** [comment]
Hey guys! Could you please let us know when it is going to be merged?

**Comment by mkendall07** [comment]
yep, should be able to get to this in a day or two.

**Comment by vodianyk** [comment]
Cool! Thanks!

**Comment by mkendall07** [comment]
cc @vodianyk ^

File: adapters/lifestreet_test.go [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+	}
+
+	fmt.Println("body", body)
+
+	req := httptest.NewRequest("POST", server.URL, body)
+	req.Header.Add("User-Agent", lsdata.deviceUA)
+	req.Header.Add("Referer", lsdata.referrer)
+	req.Header.Add("X-Real-IP", lsdata.deviceIP)
+
+	pc := pbs.ParseUIDCookie(req)
```
Comment (mkendall07) [inline adapters/lifestreet_test.go]: these also need to be refactored to latest in master. (see here for example: https://github.com/prebid/prebid-server/blob/master/adapters/appnexus_test.go#L311)

**Comment by mkendall07** [comment]
you can open a new PR with just your commits if you cannot get a clean rebase.

**Comment by Lifestreet** [comment]
See a new pull request: https://github.com/prebid/prebid-server/pull/125

**Pull request closed without merging**